server/models/Polls.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+
+const PollSchema = new mongoose.Schema({
```

Comment:
I think it's better if this is a model for an individual Poll, instead of a list. The userId doesn't have to be unique, so we can query all polls with userId == something